geoh5py/workspace/workspace.py

```diff
@@ -120,22 +120,22 @@ def active() -> Workspace:
     def _all_data(self) -> list[data.Data]:
         """Get all active Data entities registered in the workspace."""
         self.remove_none_referents(self._data, "Data")
-        return [cast("data.Data", v()) for v in self._data.values()]
+        return [cast(data.Data, v()) for v in self._data.values()]
```

Comment:
using string was known as forward declaration
Not needed anymore thanks too [PEP-563](https://www.python.org/dev/peps/pep-0563/)